core/src/test/java/com/linecorp/armeria/common/stream/ByteStreamMessageOutputStreamTest.java

```diff
@@ -151,28 +152,34 @@ void writeAfterClosed() {
     }
 
     @Test
-    void writeAfterStreamClosed() {
+    void writeAfterStreamClosed() throws InterruptedException {
+        final CountDownLatch wait = new CountDownLatch(1);
+        final CountDownLatch end = new CountDownLatch(1);
         final ByteStreamMessage byteStreamMessage = StreamMessage.fromOutputStream(os -> {
             try {
                 for (int i = 0; i < 5; i++) {
-                    if (i < 3) {
+                    if (i < 2) {
```

Comment:
```
StepVerifier
    .create(byteStreamMessage, 2)
    .expectNext(httpData(0), httpData(1))
```
oops it should be `i < 2` 😅
+ add `wait.wait(), end.wait()` to check `write()` after this `byteStreamMessage` is closed by `L179`